app/Jobs/RockTheVote/ImportRockTheVoteRecord.php

```diff
@@ -224,29 +231,52 @@ public static function shouldUpdateStatus($currentStatus, $newStatus)
     }
 
     /**
-     * Update Northstar user with record data.
+     * Get fields and values to update given user with.
      *
-     * @param NorthstarUser $user
+     * @return array
      */
-    private function updateUserIfChanged($user)
+    public function getUpdateUserPayload(NorthstarUser $user)
```

Comment:
Same here, especially with what the method used to be named was a bit confusing to me. Would maybe `getUpdatedUserData()` be clearer?

Reply:
I think that when reading it as a method name, it's not common to read it as "get updateUser() payload". Where you have another method's name in this method's name itself. It ends up sounding odd and confusing. I think with most of our method names, we aim to try and have them read as a sentence so you don't have the added need to figure out if it's referring to another method in the name itself, depending on how you read it.